release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -2399,3 +2408,66 @@ Issued: 2020-08-24
 {product-title} 4.5. Details of the update are documented in the
 link:https://access.redhat.com/errata/RHSA-2020:3520[RHSA-2020:3520] advisory.
+
+[[ocp-4-5-8]]
+=== RHBA-2020:3510 - {product-title} 4.5.8 bug fix update
+
+[[ocp-4-5-8-features]]
+==== Features
+
+[[ocp-4-5-8-added-credentialsmode-parameter-for-aws-install-workaround]]
+===== Added credentialsMode parameter to bypass faulty permissions check during AWS installation
+
+For AWS installations, {product-title} depends on an AWS policy simulator API to
+validate permissions. If an AWS account is configured to use AWS Organizations
+service control policies (SCPs), permissions are checked against the policies
+that are set in the SCPs. When SCPs include policies that use a global condition
+to deny all actions or require a specific permission, the policy simulator API
+does not correctly validate permissions. For example, policies with conditions
+such as for all regions except `us-east-1` and `us-west-2`, or for all roles
+except `role-xyz` cause the API to return false negatives. When the permissions
+cannot be validated, {product-title} AWS installations fail, even if the
+provided credentials have the required permissions to install {product-title}.
+
+With this release, you can bypass the policy simulator permissions check by
+setting a value for the `credentialsMode` parameter in the `install-config.yaml`
+configuration file.
+
+.Example `install-config.yaml` configuration file
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Mint <1>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+...
+----
+<1> This line is added to set the `credentialsMode` parameter to `Mint`.
+
+Setting a value for `credentialsMode` bypasses the permissions check for AWS
+accounts configured to use SCPs and allows the installation to proceed. When
+bypassing this check, ensure that the credentials you provide have the
+permissions that are required for the specified mode.
+
+The value of `credentialsMode` changes the behavior of the Cloud Credential
+Operator (CCO) as follows:
+
+* `Mint` - The CCO uses the provided admin-level cloud credential to run the
+installer. If the credential is not removed after installation, it is stored and
+used by the CCO to process `CredentialRequests` in the cluster and create new
+users for each with specific required permissions.
+* `Passthrough` - The CCO uses the provided non-admin cloud credential that has
+enough permissions to perform the installation to run the installer. For more
+information about locating the permissions specified in the `CredentialRequests`
+for the version of {product-title} being installed, see
+xref:../installing/installing_aws/manually-creating-iam.adoc[Manually creating IAM for AWS].
+If the credential is not removed after installation, the credential is used to
+perform all operations needed by all components in the cluster.
```

Comment:
Can drop the credential removal sentence for Passthrough, there's no point removing the root cred post-install in this mode as it's copied to every other component.